Find a functionally equivalent code:android.os.Bundle.putStringArrayList:COMMENT
Method Modifier: public      
Comment:/**
 * Inserts an ArrayList<String> value into the mapping of this Bundle, replacing
 * any existing value for the given key.  Either key or value may be null.
 *
 * @param key a String, or null
 * @param value an ArrayList<String> object, or null
 */

@@ -1,4 +1,3 @@
 {
-    unparcel();
-    mMap.put(key, value);
+    super.putStringArrayList(key, value);
 }

